draft-ietf-suit-update-management.md

```diff
@@ -156,18 +156,21 @@ The encoded versions SHOULD be semantic versions (See {{semver}}). For example,
 * 1.2-rc.3 = \[1,2,-1,3\].
 * 1.2-beta = \[1,2,-2\].
 * 1.2-alpha = \[1,2,-3\].
-* 1.2.4-alpha.1 = \[1,2,4,-3,1\].
 
 Versions SHOULD be encoded as follows:
 
 1. The first integer represents the major number. This indicates breaking changes to the component.
 2. The second integer represents the minor number. This is typically reserved for new features or large, non-breaking changes.
 3. The third integer is the patch version. This is typically reserved for bug fixes.
-4. The fourth integer is the build number.
 
-According to {{semver}}, the build number should be ignored, however this complicates the processing model. 
+The version MAY include a build number as an additional integer appended to the the three above. For example, 
```

Comment:
```suggestion
The version MAY include a build number as an additional integer appended to the three above. For example, 
```
FYI, turns out I was wrong in my previous comment on this PR in that line 159 is ok in semver (in that it follows a MAY not a SHOULD inside semver).  But the new text is still better other than the redundant "the".  [Semver](https://semver.org/) says

> Build metadata MAY be denoted by appending a plus sign and a series of dot separated identifiers immediately following the patch or pre-release version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85, 1.0.0+21AF26D3----117B344092BD

Reply:
But this means if you prefer to revert the latest change and stick with what you had originally, it's ok by me.

Reply:
I keep having to update this comment :)
Line 159 didn't follow semver because it had no plus `+`.  A third `.` can only appear later in the string after a `+`, in semver.